app/views/welcome/about.html.erb

```diff
@@ -1 +1,36 @@
-<div class='container'>
  <div class= 'about-nav'>
    <div class='about-header-image'>
      <!-- to be given an image later  -->
    </div>
    <%= link_to "About", "/about" %>
    <%= link_to "How", "/how" %>
  </div>

  <div class='about-body'>
    <h2>About She’s Coding</h2>
    <p>
      She’s Coding is an open-source website project currently under development in cooperation with the
      documentary film CODE: Debugging the Gender Gap. She’s Coding is meant to serve as the destination
      for anyone who wants to learn more about the gender gap problem in computer science and become part
      of the solution. Whether you’re a woman currently working in tech, a girl interested in learning to code,
      a male ally for women in computer science, or a company trying to find ways to increase diversity, you’ve
      come to the right place!
    </p>
    <h2>Wait, open-source?</h2>
    <p>
      Yes! We are setting up an open-source Ruby on Rails project on Github, and de- fining some
      simple rules on how to contribute. She’s Coding will be built by the community,
      for the community. The initial site will launch in mid April 2015 to coincide
      with the world premiere of the CODE documentary, and will continue to grow and
      expand in resources and functionality over time. Want to learn more or be a
      part of the she’s coding team? Contact us!
    </p>
    <p>
      Convert understanding into action – Join the She’s Coding team!
    </p>

  </div>
</div>

```

Comment:
@callapatel I don't know what editor you use, but it creates weird newlines. Which makes it impossible to use git's diff functionality on it. Maybe try out Sublime Text 2?